appveyor.yml

```diff
@@ -12,18 +12,20 @@ environment:
     - APPVEYOR_BUILD_WORKER_IMAGE: Ubuntu
       CC: clang
       CXX: clang++
+      CMAKE_ARGS: -DCMAKE_POSITION_INDEPENDENT_CODE=ON
```

Comment:
I don't think users should have to manually specify this option to get a successful build. We should be ensuring that we build enet with compatible flags when building moonlight-common-c as a shared library.